.github/actions/spell-check/expect.txt

```diff
@@ -209,23 +158,17 @@ certmgr
 cguid
 CHANGECBCHAIN
 changecursor
-Changemove
-chdir
 CHILDACTIVATE
 CHILDWINDOW
-chrdavis
-Chrzan
+Chinh
```

Comment:
This is a name. Why are we removing from names.txt and adding in expect.txt?

Reply:
No specific reason. Just wasn't recognized as a name yet.